relayer/relayer/src/config.rs

```diff
@@ -53,6 +53,24 @@ pub struct Config {
     )]
     pub run_trader_component: bool,
 
+    /// Trader component will always keep this + current base_fee in the guardians balance and will only sell the surplus
+    ///
+    /// Defaults to 100 AZERO
+    #[arg(long, default_value = "100_000_000_000_000")]
```

Comment:
Let's set the default value to 1000 AZERO

Reply:
ok